libherokubuildpack/src/buildpack_output/style.rs

```diff
@@ -0,0 +1,28 @@
+//! Helpers for formatting and colorizing your output.
+
+use crate::buildpack_output::ansi_escape;
+
+/// Decorate a URL for the build output.
+pub fn url(contents: impl AsRef<str>) -> String {
+    ansi_escape::inject_default_ansi_escape(ansi_escape::CYAN, contents)
+}
+
+/// Decorate the name of a command being run i.e. `bundle install`.
+pub fn command(contents: impl AsRef<str>) -> String {
+    value(ansi_escape::inject_default_ansi_escape(
+        ansi_escape::BOLD_CYAN,
+        contents.as_ref(),
```

Comment:
(the `body` argument to `inject_default_ansi_escape` is of type `impl AsRef<str>`, making the `as_ref()` here redundant)

```suggestion
        contents,
```